### PR TITLE
Throw an Exception instead of using exit

### PR DESCRIPTION
**Pull request by Chemaclass**
# Description

Getting rid of the `exit` keyword in Main.

# Changes

- Removed exit's from `Main`. Using an `Exception` instead.
- Clean up `TestCommand`.

### CODE REVIEW

File: src/php/Commands/TestCommand.php [hunk 1]
```diff
@@ -24,23 +24,18 @@ public function __construct(?Runtime $runtime = null)
...
     {
-        if (count($paths) === 0) {
-            $namespaces = $this->getNamespacesFromConfig($currentDirectory);
-        } else {
-            $namespaces = [];
-            foreach ($paths as $filename) {
-                $namespaces[] = CommandUtils::getNamespaceFromFile($filename);
-            }
-        }
+        $namespaces = count($paths) === 0
```
Comment (JesusValera) [inline src/php/Commands/TestCommand.php:27]: Better using `empty()` than `count($var) === 0` doesn't it?

File: src/php/Commands/TestCommand.php [hunk 2]
```diff
@@ -60,28 +55,20 @@ private function getNamespacesFromConfig(string $currentDirectory): array
 
     private function initializeRuntime(string $currentDirectory): Runtime
     {
-        if ($this->runtime === null) {
-            $rt = CommandUtils::loadRuntime($currentDirectory);
-        } else {
-            $rt = $this->runtime;
-        }
+        $rt = $this->runtime ?? CommandUtils::loadRuntime($currentDirectory);
```
Comment (JesusValera) [inline src/php/Commands/TestCommand.php:58]: What about renaming `$rt` to `$runtime`? :)

**Review by JesusValera — APPROVED** [review]
Nice PR, I think is smart the exception than the `exit()` inside the program logic 💃🏼

File: src/php/Commands/TestCommand.php [hunk 3]
```diff
@@ -44,21 +44,21 @@ private function getNamespacesFromConfig(string $currentDirectory): array
         $config = CommandUtils::getPhelConfig($currentDirectory);
         $namespaces = [];
 
-        if (isset($config['tests'])) {
-            foreach ($config['tests'] as $testDir) {
-                $namespaces = array_merge($namespaces, $this->findAllNs($currentDirectory . $testDir));
-            }
+        $testDirectories = $config['tests'] ?? [];
```
Comment (JesusValera) [inline src/php/Commands/TestCommand.php:47]: 👍🏼 👍🏼

**Review by JesusValera — APPROVED** [review]

**Review by jenshaase — APPROVED** [review]
Super nice!

**Pull request merged**